classes/OpenXdmod/Setup/ResourcesSetup.php

```diff
@@ -123,26 +123,41 @@ public function addResource(array $resource)
         // Look up the resource type id for the string that was entered
 
         $availableTypes = XdmodConfiguration::assocArrayFactory('resource_types.json', CONFIG_DIR)['resource_types'];
+        $availableResourceAllocationTypes = XdmodConfiguration::assocArrayFactory('resource_allocation_types.json', CONFIG_DIR)['resource_allocation_types'];
 
         $typeAbbrev = 'UNK';
+        $resourceAllocationTypeAbbrev = 'UNK';
+
         foreach($availableTypes as $abbrev => $type) {
             if (strtolower($abbrev) === $resource['type']) {
                 $typeAbbrev = $abbrev;
                 break;
             }
         }
 
+        foreach($availableResourceAllocationTypes as $abbrev => $type) {
+            if (strtolower($abbrev) === $resource['resource_allocation_type']) {
+                $resourceAllocationTypeAbbrev = $abbrev;
+                break;
+            }
+        }
+
         $this->resources[] = array(
-            'resource'         => $resource['resource'],
-            'resource_type' => $typeAbbrev,
-            'name'             => $resource['name'],
+            'resource'                  => $resource['resource'],
+            'resource_type'             => $typeAbbrev,
+            'resource_allocation_type'  => $resourceAllocationTypeAbbrev,
+            'name'                      => $resource['name'],
```

Comment:
```suggestion
            'name'                      => $resource['name'],
            'resource_allocation_type'  => $resourceAllocationTypeAbbrev,
```
This changes to match the ordering used by the migration script (perhaps the other way around is preferable?).